ansible/roles/base/tasks/main.yml

```diff
@@ -17,18 +15,23 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
+      apt: 
```

Comment:
I’m not sure why, but your editor is forcing a bunch of these to be multi-line for no reason? It’s also continuing to remove spaces. It makes it look like there are changes where there aren’t. There are certainly some in here, but it’s hard to keep track of them.